x/ccv/consumer/keeper/validators.go

```diff
@@ -102,32 +107,41 @@ func (k Keeper) ValidatorByConsAddr(sdk.Context, sdk.ConsAddress) stakingtypes.V
 	return stakingtypes.Validator{}
 }
 
+// Calls SlashWithInfractionReason with Infraction_INFRACTION_UNSPECIFIED.
+// ConsumerKeeper must implement StakingKeeper interface.
+// This function should not be called anywhere
+func (k Keeper) Slash(ctx sdk.Context, addr sdk.ConsAddress, infractionHeight, power int64, slashFactor sdk.Dec) math.Int {
+	return k.SlashWithInfractionReason(ctx, addr, infractionHeight, power, slashFactor, stakingtypes.Infraction_INFRACTION_UNSPECIFIED)
+}
+
 // Slash queues a slashing request for the the provider chain
 // All queued slashing requests will be cleared in EndBlock
-func (k Keeper) Slash(ctx sdk.Context, addr sdk.ConsAddress, infractionHeight, power int64, slashFactor sdk.Dec, infraction stakingtypes.InfractionType) {
-	if infraction == stakingtypes.InfractionEmpty {
-		return
+// Called by Slashing keeper in SlashWithInfractionReason
+func (k Keeper) SlashWithInfractionReason(ctx sdk.Context, addr sdk.ConsAddress, infractionHeight, power int64, slashFactor sdk.Dec, infraction stakingtypes.Infraction) math.Int {
+	if infraction == stakingtypes.Infraction_INFRACTION_UNSPECIFIED {
+		return math.NewInt(0)
 	}
 
 	// If this is a previously standalone chain and infraction happened before the changeover was completed,
 	// slash only on the standalone staking keeper.
 	if k.IsPrevStandaloneChain(ctx) && infractionHeight < k.FirstConsumerHeight(ctx) {
-		k.standaloneStakingKeeper.Slash(ctx, addr, infractionHeight, power, slashFactor, stakingtypes.InfractionEmpty)
-		return
+		// NOTE: I'm not sure this code is 100% correct and it's relatively newly implemented
```

Comment:
This is because the Slash function for a standalone chain doesn't require an infraction type. See the staking module.

Reply:
changed comment to:
`// Slash for a standalone chain does not require an infraction reason so we pass in Infraction_INFRACTION_UNSPECIFIED
`